ksp_plugin/orbit_analyser.cpp

```diff
@@ -82,8 +83,9 @@ Status OrbitAnalyser::RepeatedlyAnalyseOrbit() {
     {
       absl::MutexLock l(&lock_);
       if (!guarded_parameters_.has_value()) {
-        // No parameters, let's wait for them to appear.
-        continue;
+        // No parameters, stop.
+        analyser_done_ = true;
+        return Status::OK;
```

Comment:
If you detach, you don't need to change anything here, it will ultimately land on a `RETURN_IF_STOPPED`.

Reply:
Rewrote as discussed.